modules/dashboard/php/dashboard.class.inc

```diff
@@ -268,34 +268,36 @@ class Dashboard extends \NDB_Form
                  ORDER BY Testdate DESC LIMIT 4",
                 array()
             );
-            for ($i=0; $i < sizeof($bvl_feedback); $i++) {
-                if ($bvl_feedback[$i]['Testdate'] > $last_login) {
-                    $bvl_feedback[$i]['new'] = 1;
+            $frontend_feedback = array();
+            foreach ($bvl_feedback as $row) {
```

Comment:
```suggestion
            foreach ($bvl_feedback as &$row) {
```

The PHP documentation (https://www.php.net/manual/en/control-structures.foreach.php) implies that foreach passes by value and `&` is needed to make it by reference. Since this is mutating `$row`, it should be a reference to ensure the behaviour remains the same.

Reply:
`$bvl_feedback` is just a DB fetch that gets added to the front-end. `$frontend_feedback` keeps track of what gets modified in the for loop and then that array gets passed to the tpl data.

Why do you want to use references here? Not that it wouldn't work, but it's really unorthodox for a pretty routine LORIS thing.